notebooks/multiagent_web_assistant.ipynb

```diff
@@ -0,0 +1,1407 @@
+{
```

Comment:
<p>you can move this up one code cell IMO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>